packages/suite/src/components/suite/modals/AddAccount/index.tsx

```diff
@@ -77,21 +81,37 @@ export const AddAccount = ({ device, onCancel, symbol, noRedirect }: AddAccountP
         (n: Network) => !(n.symbol === 'regtest' && !debug.showDebugMenu),
     );
 
-    const handleNetworkSelection = (symbol?: Network['symbol']) => {
-        if (symbol) {
-            const networkToSelect = NETWORKS.find(n => n.symbol === symbol);
+    const handleNetworkSelection = useCallback(
```

Comment:
useCallback not needed

Reply:
02f7f6de87283cd2290b0e3c80f7028c38846769

Reply:
also remove import of useCallback as it is no more used in this file